tests/lib/rules/lines-around-comment.js

```diff
@@ -317,8 +317,195 @@ ruleTester.run("lines-around-comment", rule, {
                 allowBlockEnd: true
             }],
             ecmaFeatures: {classes: true}
-        }
+        },
+
+        // check for object start comments
```

Comment:
Here is an example: https://github.com/eslint/eslint/blob/master/tests/lib/rules/indent.js